rules/ast/functions.go

```diff
@@ -291,6 +291,25 @@ func dropCommonLabelsImpl(timestamp clientmodel.Timestamp, args []Node) interfac
 	return vector
 }
 
+// === round(vector VectorNode) Vector ===
+func roundImpl(timestamp clientmodel.Timestamp, args []Node) interface{} {
+	// round to even
```

Comment:
Yep, that sounds great! And LGTMs are usually written :+1: in non-Google-land ;)